### PR TITLE
[Adama Ebenezer]-sage-Solidity Marketplace Contract for Furniture Transactions on Celo Blockchain

### PR DESCRIPTION
**Pull request by Adam-eben**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+- Familiarity with the JavaScript programming language, as it will help you understand the concepts and syntax used in the code.
+
+- A basic understanding of blockchain technology and how it works, including concepts like smart contracts and decentralized applications.
+
+- Basic knowledge of the Solidity programming language, which is used for developing smart contracts on the Ethereum blockchain. This includes understanding data types, functions, and contract deployment.
+
+Having these prerequisites will enable you to grasp the concepts and follow the steps in this tutorial more effectively. However, even if you are new to these topics, the tutorial will provide explanations and guidance to help you understand and learn along the way.
+
+To start this tutorial, we will utilize the Remix IDE as our development environment to write the smart contract. Let's begin the process by setting up Remix and creating our contract.
+
```
Comment (4undRaiser) [inline blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md:43]: headline missing

File: blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+Throughout the tutorial, we covered key concepts such as smart contracts, data structures, mappings, modifiers, and transaction handling. By following this tutorial, you now have a solid understanding of how to develop a decentralized marketplace on the Celo blockchain.
+
+Building decentralized marketplaces opens up new opportunities for secure and transparent transactions without relying on intermediaries. It empowers users to directly engage in peer-to-peer exchanges while maintaining control over their assets. With the knowledge gained from this tutorial, you can further explore and enhance the functionality of your marketplace or embark on other blockchain development projects.
+
+## NEXT STEP
+
+Great job! It's always helpful to provide additional resources for further learning. Don't hesitate to reach out if you have any more questions or if you need further assistance. You can also connect with me on Discord @adammm#0518 if you have any follow-up questions or want to continue the conversation. Happy learning and best of luck with your decentralized marketplace project!
```
Comment (4undRaiser) [inline blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md:420]: you mention additional resource here but i can't find any

File: blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+I'm Adama Ebenezer, a web3 developer from Nigeria with a strong passion for blockchain technology. I'm excited to leverage my skills and knowledge to contribute to the blockchain space.
+
+
+
+
+
+
+
+
+
```
Comment (4undRaiser) [inline blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md:433]: pls remove unnecessary spaces here

File: blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+**Struct**:
+
+- `FurnitureData`: Defines a structure that represents a furniture item in the marketplace. It includes properties such as the owner's address, image, description, edition, size, and price.
+By defining the MarketPlace contract and its associated state variables and struct, we establish the foundation for managing furniture items and facilitating transactions within the decentralized marketplace.
+
+## Step 4: Implement the Furniture Mapping
+
+```solidity
+  }
```
Comment (4undRaiser) [inline blog/05-3-2023-Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain/Solidity-Marketplace-Contract-for-Furniture-Transactions-on-Celo-Blockchain.md:229]: remove curly bracket above the mapping  here

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**